neuralpredictors/layers/readouts/factorized.py

```diff
@@ -19,12 +23,31 @@ def __init__(
         init_noise=1e-3,
         constrain_pos=False,
         positive_weights=False,
+        positive_spatial=True,
         shared_features=None,
         mean_activity=None,
         spatial_and_feature_reg_weight=None,
-        gamma_readout=None,  # depricated, use feature_reg_weight instead
+        gamma_readout=None,
         **kwargs,
     ):
+        """
+
+        Args:
+            in_shape: batch, channels, height, width (batch could be arbitrary)
+            outdims: number of neurons to predict
+            bias: if True, bias is used
+            normalize: if True, normalizes mask using by l2 norm
```

Comment:
I think - if I am not mistaken - this only affects the spatial mask, so I would suggest to be precise here for users that do not know the code yet in detail
```suggestion
            normalize: if True, normalizes the spatial mask using l2 norm
```